Feature/PRLC100-209

giorgi-hmcts opened the pull request:
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRLC100-209

### Change description ###
- Add content to `c100-rebuild/contact-representative` page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

yogendraU commented on src/main/steps/c100-rebuild/contact-representative/content.ts:
```diff
@@ -0,0 +1,56 @@
+export const form: FormContent = {
+  fields: {},
+  submit: {
+    text: () => 'Close the application',
```
Can we move this to variables

giorgi-hmcts replied:
Moved inside variables, all tests passing ✅

channa-patil reviewed (changes requested):
@giorgi-hmcts Can we move 'contact-representative' folder inside 
src/main/steps/c100-rebuild/screening-questions as this is part of screening questions feature. Also please append /screening-questions in the URL

giorgi-hmcts commented:
> @giorgi-hmcts Can we move 'contact-representative' folder inside src/main/steps/c100-rebuild/screening-questions as this is part of screening questions feature. Also please append /screening-questions in the URL

Moved 'contact-representative' inside inside 'src/main/steps/c100-rebuild/screening-questions' and updated the URL ✅
